### PR TITLE
Added Modal

### PR DESCRIPTION
**Pull request by HrishikeshMahalle**

### CODE REVIEW

File: Components-JS/modal.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+//  function lateRun(){
+//     window.addEventListener('click',(e)=>{
+//         if(document.querySelector('.act-box').contains(e.target)){
+//         console.log("cliked inside")
+//         }else{
+//             removeClass()
+//         }
+//     })
+//  }
+
```
Comment (Hamza-777) [inline Components-JS/modal.js:14]: Please remove commented code if not needed.

**Review by Hamza-777 — APPROVED** [review]
Looks good otherwise.

File: ComponentsCss/code.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.code-box {
+    /* max-width: 75vw; */
```
Comment (md-amir-gauhar) [inline ComponentsCss/code.css:2]: remove the commented code to make it look clean

File: Modal/modal.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+&lt;<span class="hljs-built_in">div</span> class=<span class="hljs-string">"modal-box"</span>&gt;<span class="hljs-string">
+  &lt;<span class="hljs-built_in">div</span> class=<span class="hljs-string">"modal"</span>&gt;<span class="hljs-string"> 
+    &lt;<span class="hljs-built_in">div</span> class=<span class="hljs-string">"modal-head"</span>&gt;<span class="hljs-string">
+      &lt;h1&gt;Heading&lt;/h1&gt;
+      &lt;p&gt;Heading&lt;/p&gt;
+    </span></span> &lt;/<span class="hljs-built_in">div</span>&gt;
+    &lt;<span class="hljs-built_in">div</span> class=<span class="hljs-string">"modal-buttons"</span>&gt;<span class="hljs-string">
+      &lt;<span class="hljs-built_in">button</span> class=<span class="hljs-string">"actions"</span>&gt;<span class="hljs-string"> Action1</span> &lt;/<span class="hljs-built_in">button</span>&gt;
+      &lt;<span class="hljs-built_in">button</span> class=<span class="hljs-string">"actions1"</span>&gt;<span class="hljs-string"> Action2</span> &lt;/<span class="hljs-built_in">button</span>&gt;
```
Comment (md-amir-gauhar) [inline Modal/modal.html:95]: Use some code formatter like prettier

**Review by md-amir-gauhar — APPROVED** [review]

File: Components-JS/modal.js (hunk 1 above)
Comment (HrishikeshMahalle) [reply Components-JS/modal.js:14]: sure hamza

File: ComponentsCss/code.css (hunk 2 above)
Comment (HrishikeshMahalle) [reply ComponentsCss/code.css:2]: Yes i'll try my best

**Pull request merged**